cmdutils/switches/switches.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 OnMetal authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package switches provides new type that implements flag.Value interface -- Switches.
+// It can be used for enabling/disabling controllers/webhooks in your controller manager.
+package switches
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/kustomize/kyaml/sets"
+)
+
+const (
+	DefaultValue = "*"
```

Comment:
`DefaultValue` doesn't tell what's about to happen - should be called `switches.All`.